tests/test_physics.py

```diff
@@ -593,14 +593,9 @@ def getRestPositions(articulated_object):
 
 def getRandomPositions(articulated_object):
     r"""Constructs a random pose vector for an ArticulatedObject with unit quaternions for spherical joints."""
-    lower_limits = articulated_object.get_joint_position_limits(upper_limits=False)
-    upper_limits = articulated_object.get_joint_position_limits(upper_limits=True)
-    for ix, val in enumerate(lower_limits):
-        if val < -1:
-            lower_limits[ix] = -1
-    for ix, val in enumerate(upper_limits):
-        if val > 1:
-            upper_limits[ix] = 1
+    joint_limits = articulated_object.joint_position_limits
```

Comment:
```suggestion
    lower_limits, upper_limits = articulated_object.joint_position_limits
```

Reply:
Is this more efficient given the lines below with limiting?

Reply:
It will just error if we ever increase or decrease the number of elements returned by the function and is useful for that purpose. Also means you don't have to worry about index out of bound accesses for the tuple.